### ISSUE TITLE
Outdated contact information on covjson.org

### ISSUE THREAD

**Issue by ways** [opengeospatial/CoverageJSON#171]
Hi

I tired mailing the project via links on https://covjson.org/, but both addresses bounced.

The website has a dead link: https://covjson.org/domain-types/. It is referred to both from https://covjson.org/cookbook/ and from https://www.w3.org/TR/covjson-overview/ and should be fixed, please.

**Comment by jonblower** [comment]
Good point, thank you for reporting this! We should probably just remove the email links as this GitHub repo is the best way to contact the community.

@letmaik what's the best way of fixing the dead link? Should we set up a redirect to the new spec location?

**Comment by jonblower** [comment]
Update: I've (finally) removed the outdated contact information on https://covjson.org. I've edited the source of the cookbook, but I can't figure out how to deploy the updated version. (@letmaik, any pointers?)

**Comment by chris-little** [comment]
@jonblower @letmaik Can I close this issue yet please?
